packages/frontend-app-api/api-report.md

```diff
@@ -15,4 +16,33 @@ export function createApp(options: {
 }): {
   createRoot(): JSX.Element;
 };
+
+// @public (undocumented)
+export function createExtensionTree(): ExtensionTree;
+
+// @public (undocumented)
+export interface ExtensionInstance {
+  // (undocumented)
+  readonly $$type: '@backstage/ExtensionInstance';
+  readonly attachments: Map<string, ExtensionInstance[]>;
+  getData<T>(ref: ExtensionDataRef<T>): T | undefined;
+  // (undocumented)
+  readonly id: string;
+}
+
+// @public (undocumented)
+export interface ExtensionTree {
+  // (undocumented)
+  getExtension(id: string): ExtensionInstance | undefined;
+  // (undocumented)
+  getExtensionAttachments(id: string, inputName: string): ExtensionInstance[];
+}
+
+// @public (undocumented)
+export interface ExtensionTreeNode {
```

Comment:
This one really didn't need to be exported right? It's not used anywhere at least yet

Reply:
Ah so, we should keep this public but hide ExtensionInstance instead